easybuild/easyconfigs/h/HEALPix/HEALPix-3.82-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'MakeCp'
+
+name = 'HEALPix'
+version = '3.82'
+
+homepage = 'http://healpix.sourceforge.net/'
+description = """Hierarchical Equal Area isoLatitude Pixelation of a sphere."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
```

Comment:
`HEALPix` has Fortran `.mod` files (in `include` directory), so should be at `GCC` and not `GCCcore`. See https://github.com/easybuilders/easybuild-framework/issues/4203